packages/react-ui/src/features/forms/components/ap-form.tsx

```diff
@@ -0,0 +1,240 @@
+                                  htmlFor={input.displayName}
+                                  className="flex items-center justify-between"
+                                >
+                                  {input.displayName}
+                                </FormLabel>
+                                <FormControl className="flex flex-col gap-1">
+                                  <>
+                                    {input.type === FormInputType.TEXT_AREA && (
+                                      <Textarea
+                                        onChange={field.onChange}
+                                        value={
+                                          field.value as string | undefined
+                                        }
+                                      />
+                                    )}
+                                    {input.type === FormInputType.TEXT && (
+                                      <Input
+                                        onChange={field.onChange}
+                                        value={
+                                          field.value as string | undefined
+                                        }
+                                      />
+                                    )}
+                                    {input.type === FormInputType.FILE && (
+                                      <Input
+                                        onChange={(e) => {
+                                          const file = e.target.files?.[0];
+                                          if (file) {
+                                            fileToBase64(file, (result) => {
+                                              field.onChange(result);
+                                            });
+                                          }
+                                        }}
+                                        placeholder={input.displayName}
+                                        type="file"
+                                      />
+                                    )}
+                                    <ReadMoreDescription
+                                      text={input.description ?? ''}
+                                    />
+                                  </>
+                                </FormControl>
+                              </FormItem>
+                            )}
+                          </>
+                        )}
+                      />
+                    );
+                  })}
+                </div>
+                <Button type="submit" className="w-full" loading={isPending}>
+                  Submit
+                </Button>
+
+                {markdownResponse && (
+                  <>
+                    <Separator className="my-4" />
+                    <ApMarkdown markdown={markdownResponse} />
+                  </>
+                )}
+              </CardContent>
```

Comment:
The powered by activepieces section is missing.
![image](https://github.com/user-attachments/assets/ddabd951-d640-4bb2-889b-24689a629ec4)

![image](https://github.com/user-attachments/assets/f41d85b5-283e-4520-8e15-d24444c808c7)

Reply:
Opened a ticket on the react project